mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,17 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(happy) {
+  if (happy == true) {
```

Comment:
Since booleans can only be true or false you shouldn't need to check if happy is equal to true or false. Checking the variable happy should be enough to determine whether the mood is happy or not.

You can read more on booleans here: https://www.w3schools.com/js/js_booleans.asp